src/pages/block/Block.tsx

```diff
@@ -199,50 +182,37 @@ const Block: React.FC<Props> = (props: Props) => {
               <div className="detail-tile script-tile">
                 <div className="script-label-and-copy-row">
                   <label>INVOCATION SCRIPT</label>{' '}
-                  {chain === 'neo2' ? (
-                    <Copy text={block ? block.script?.invocation : ''} />
-                  ) : (
-                    <Copy
-                      text={
-                        block && block.witnesses
-                          ? block.witnesses[0].invocation
-                          : ''
-                      }
-                    />
-                  )}
+                  <Copy
+                    text={
+                      block && block.witnesses
+                        ? block.witnesses[0].invocation
+                        : ''
+                    }
+                  />
                 </div>
                 <span>
-                  {chain === 'neo2'
-                    ? !isLoading && block && block.script?.invocation
-                    : !isLoading &&
-                      block &&
-                      block.witnesses &&
-                      block.witnesses[0].invocation}
+                  {!isLoading &&
+                    block &&
+                    block.witnesses &&
+                    block.witnesses[0].invocation}
```

Comment:
I know this is probably annoying feedback but we dont actually need the `&&` here to access this value safetly if we are using optional chaining... Pretty sure this can just be something like this:

```javascript
!isLoading && block?.witnesses?[0].invocation
```

Reply:
I could not get this to work with that minimized version you proposed, keeps complaining with
> Property 'invocation' does not exist on type 'number[]'

For some reason this is the best that did work
```
!isLoading &&
                    block?.witnesses &&
                    block.witnesses[0].invocation
```